pom.xml

```diff
@@ -83,7 +83,14 @@
         <version>5.2.8.RELEASE</version>
         <scope>core</scope>
     </dependency>
-      
+
+    <!-- https://mvnrepository.com/artifact/commons-io/commons-io -->
+    <dependency>
+        <groupId>commons-io</groupId>
```

Comment:
Please don't add the library if there's no code using it.
Maybe you forgot to add Java code to this pull request?

Reply:
I forgot to add this change to previous PR.
I should add it separately from this changes, I think.